### DISCUSSION TITLE
Publisher verification unknown

### DISCUSSION THREAD

**Discussion by ArronKnight** [airsdk/Adobe-Runtime-Support#1966] [General]
Hi, 
is there any update on when the signing certificate issue will likely be fixed?
![image](https://user-images.githubusercontent.com/107183555/173016991-98fe2ebd-8f6b-4ec0-83dc-fa2d69d1b071.png)

Thanks

**Comment by ajwfrost** [comment]
Hi @ArronKnight  - I've just had someone look back into this. A lot of the security and certificate handling had already been added back into the runtime, but this is lacking one piece still which means we don't get a full trust path to the signing certificate and hence you get the dialog per that screenshot.

The missing piece is to integrate into the operating system and validate the root certificate.. we've just done a quick test where we hacked the code to pretend that the root cert was trusted by the OS, and the publisher information was shown properly. So there's a bit of work to do for this, but it should be something we can address during the next month and would therefore be in our next release of the shared runtime.

thanks
